src/seedu/addressbook/commands/CheckCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.commands;
+
+/**
+ * Gives the size of the address book - number of persons.
```

Comment:
Does this class give the size of the address book? 
I think it just checks whether persons are there in the address book or not.